docs/2-deployment/email-notification-scheduler.md

```
@@ -600,6 +635,7 @@ gcloud run jobs execute email-weekly-digest-job --region ${REGION}
 
 | 郵件任務 | 頻率 | Cron | 執行時間（台北） | 預估執行時長 |
 | -------- | ---- | ---- | --------------- | ----------- |
+| **訂閱通知** | 每日 | "0 10 * * *" | 每日 10:00 | ~2 分鐘 |
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)